contracts/FlowIDTableStaking.cdc

```diff
@@ -1636,12 +1667,14 @@ pub contract FlowIDTableStaking {
 
     /// Adds a node and/or a delegator to the list of node IDs who have pending token movements
     /// or whose delegators have pending movements
-    access(contract) fun setNewMovesPending(nodeID: String, delegatorID: UInt32?) {
+    /// Can pass an existing list to add it to that instead of the saved one, like during `moveTokens`
+    access(contract) fun setNewMovesPending(nodeID: String, delegatorID: UInt32?, existingList: {String: {UInt32: Bool}}?): {String: {UInt32: Bool}} {
         if self.nodes[nodeID] == nil {
-            return
+            return existingList ?? {}
         }
-        let movesPendingList = self.account.load<{String: {UInt32: Bool}}>(from: /storage/idTableMovesPendingList)
-            ?? panic("No moves pending list in account storage")
+
+        let movesPendingList = existingList ?? (self.account.load<{String: {UInt32: Bool}}>(from: /storage/idTableMovesPendingList)
+            ?? panic("No moves pending list in account storage"))
```

Comment:
are the outer parenthesis needed?

Reply:
Probably not, but it felt better to put them there just to be clear